setup/kubernetes/local-preview.md

```diff
@@ -130,7 +128,7 @@ rebinding protection. Here's how:
    sudo apt-get install dnsmasq
    ```
 
-1. Create dnsmasq configuration for the .coder domain
+1. Create a dnsmasq configuration for the `.coder` domain
```

Comment:
One more option is to modify `/etc/hosts` to manually add records pointing to localhost, it's not as seamless as dnsmasq but will work on Windows too https://en.wikipedia.org/wiki/Hosts_(file)#Location_in_the_file_system

Reply:
Would this be for each dev url added?

Reply:
Yeah, the hosts files don't support wildcards, so a user would have to manually add like

```
abc.coder.localhost 127.0.0.1
def.coder.localhost 127.0.0.1
```

where they configure their Dev URL suffix domain as `coder.localhost`. admittedly this is a pretty bad user experience.

the option with nip is easier to set up but the URLs are uglier :man_shrugging:

Reply:
Yeah, would prefer not to mention this for now